test/unittests/execute_control_test.cpp

```diff
@@ -480,6 +480,51 @@ TEST(execute_control, br_1_out_of_function_and_imported_function)
     EXPECT_EQ(ret[0], 1);
 }
 
+TEST(execute, br_table)
+{
+    /*
+    (func (param i32) (result i32)
+      (block
+        (block
+          (block
+            (block
+              (block
+                (br_table 3 2 1 0 4 (get_local 0))
+                (return (i32.const 99))
+              )
+              (return (i32.const 100))
+            )
+            (return (i32.const 101))
+          )
+          (return (i32.const 102))
+        )
+        (return (i32.const 103))
+      )
+      (i32.const 104)
+    )
+   */
+    const auto bin = from_hex(
+        "0061736d0100000001060160017f017f030201000a330131000240024002"
+        "400240024020000e04030201000441e3000f0b41e4000f0b41e5000f0b41"
+        "e6000f0b41e7000f0b41e8000b000c046e616d6502050100010000");
+
+    for (const auto param : {0u, 1u, 2u, 3u, 4u})
```

Comment:
I would add one more input value for default label with value other than `4u`.

Reply:
This I test below outside of the loop, if I understand you correctly.

Reply:
Oh yes, right. But I think you are overusing value `42`. The value `5` would be better because it is on the edge of being out of table index.

Reply:
Ok I'll add another case to the loop then